cluster/ci/monitoring/deck.json

```diff
@@ -76,7 +76,7 @@
                "refId": "C"
             },
             {
-               "expr": "histogram_quantile(0.5, sum(rate(deck_http_request_duration_seconds_bucket{path=~\"/tide.*\", method=~\"${method}\", status=~\"${status}\"}[5m])) by (le))",
+               "expr": "histogram_quantile(0.5, sum(rate(deck_http_request_duration_seconds_bucket{path=~\"/prowjobs.js.*\", method=~\"${method}\", status=~\"${status}\"}[5m])) by (le))",
```

Comment:
This doesn't need to be a regex, right?

Reply:
Correct, it does not in general.
We use regex here (and in other targets too) because in some special case ... `/prowjobs.js?1=3` and the path will be like that. Maybe I am crazy.

OhOh ... we do prefixing now. ^_^
Then in principle, those regex should still work.

Let me remove regex in all places and add `others` tomorrow.

Reply:
Done.